### DISCUSSION TITLE
Custom README name?

### DISCUSSION THREAD

**Discussion by ngdangtu-vn** [npm/rfcs#807] [RRFC]
I have a monorepo and a plan to have multiple workspace package inside. So it will be a problem if all the workspace package has the same README name. So I make a pattern name like this: `package-name.note.md` as a way to distinguish the README. However NPM doesn't support this. Can I propose a field in `package.json` to indicate which MD file should be treated as README?

```json
"repository": {
    "type": "git",
    "url": "git+https://gitlab.com/ndt-deps/astro-pkg.git",
    "directory": "packages/loaders/toml",
    "readme": "toml.note.md"
},
```

While there is a way to work around the problem, but I don't see any reason NPM shouldn't support this. My apology if we have one reason and I didn't know.

**Comment by ljharb** [comment]
Why would it be a problem? jump-to-file works fine with lots of same-named files in different folders.

**Reply by ngdangtu-vn** [reply]
If you have 10 readme files and you want to jump to readme in kitten directory, isn't it a pain? Moreover, when you have a lot of tabs open, so much that its size compress to only fit a few letter of the file name. Now, can you distinguish which file is readme file in doggy dir and which is from kitten dir?

**Reply by ljharb** [reply]
i don't see why - you'd type "readme kitten" and it'd put it at the top.

i agree on the tab name part, but i believe vscode does a pretty good job of showing you the parts that differ, to handle that exact case.

**Reply by ngdangtu-vn** [reply]
Well, you are correct about the jump but between "kitten↵" and "readme kitten↵" I would prefer the shorter choice.

About showing path name, in most of the theme it's usually the dim color. That is right design, but it also not very useful when you have a lot same name files and the only different got a bit hard to read. I don't like to fix the theme for this case because the theme is good as it is and the path should not response for telling us the different between files like that.

I see this problem is the same with `index.html`.
